createPR.js

```diff
@@ -1,15 +1,27 @@
+/*global process*/
 const simpleGit = require("simple-git/promise");
 const { Octokit } = require("@octokit/rest");
 const fs = require("fs");
 
 const issue_number = process.argv[2];
 
+// the following resource will be applied to the cwd (current working directory)
 const git = simpleGit(".");
-const configFile = fs.readFileSync("./hackattoni.json", "utf8");
-const {github_token, remote, fork} = JSON.parse(configFile);
-const octokit = new Octokit({ auth: github_token });
+const configOptions = fs.readFileSync("./configOptions.json", "utf8");
```

Comment:
I think configOptions.json is not very meaningful as a name, it should have a name that reminds its purpose. Remember that you should put this file in your project repo. Think of other configuration files for tools, e.g. babel.config.json.

Reply:
right and i spent 10 minutes thinking, rethinking and renaming it :)
alternatives
- [ ] createPROptions.json
- [ ] create_PR_Options.json
- [ ] automatic_create_PR_options.json
- [ ] hackattoni.json (didn't like much this)
- other:

Reply:
what about the name?

Reply:
Thinking out loud: maybe we should just choose a name for this project and then use <projectName>.json

Some project names coming to my mind:
 - [ ] git-workflow-tools
 - [ ] automata
 - [ ] just-do-it